backend/src/starknet.ts

```diff
@@ -0,0 +1,26 @@
+import { StarknetIdNavigator } from "starknetid.js";
+import { RpcProvider, constants } from "starknet";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const NODE_URL: string = process.env.NODE_URL || "";
+
+export const getStarknetId = async (
+  address: string
+): Promise<string | null> => {
+  const provider = new RpcProvider({
+    nodeUrl: NODE_URL,
+    chainId: constants.StarknetChainId.SN_MAIN
+  });
+
+  const starknetIdNavigator = new StarknetIdNavigator(
+    provider,
+    constants.StarknetChainId.SN_MAIN
+  );
+
+  const domain = await starknetIdNavigator.getStarkName(address);
+  const id = await starknetIdNavigator.getStarknetId(domain);
+
+  return id;
+};
```

Comment:
getStarkName throws an error and therefore kills the whole backend when the starknetId hasn't been found. Please reach out to me on Telegram too.

Reply:
Alright

Reply:
@tensojka I've fixed the issue causing the backend to fail when `getStarkname()` fails to get the address' domain. Please take a look.